projects/configs/tag-operations/uclh-nasogastric-tube-project-ngt-only-full-dataset.yaml

```diff
@@ -375,6 +375,10 @@
   group: 0x0020
   element: 0x0020
   op: "keep"
+- name: "Patient Orientation Code Sequence"
```

Comment:
this and the other tags should be in order

Reply:
YAML file has been re-ordered here [33d413e](https://github.com/SAFEHR-data/PIXL/pull/493/commits/33d413ef3b1d0d68d8ed9ed8eabb791702ea560e)